friendshipper/src/routes/servers/+page.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+	import { Button, Card, Spinner } from 'flowbite-svelte';
+	import { emit } from '@tauri-apps/api/event';
+	import { onMount } from 'svelte';
+	import { CirclePlusOutline, RefreshOutline } from 'flowbite-svelte-icons';
+	import { get } from 'svelte/store';
+	import { appConfig, builds, selectedCommit } from '$lib/stores';
+	import type { ArtifactEntry, GameServerResult, Nullable, SyncClientRequest } from '$lib/types';
+	import ServersListTable from '$lib/components/servers/ServersListTable.svelte';
+	import { getServers } from '$lib/gameServers';
+	import ServerModal from '$lib/components/servers/ServerModal.svelte';
+	import { syncClient } from '$lib/builds';
+
+	let fetchingServers = false;
+	let servers: GameServerResult[] = [];
+
+	// create server modal
+	let showModal = false;
+	let selected: Nullable<ArtifactEntry> = get(selectedCommit);
+
+	const updateServers = async () => {
+		if ($appConfig.initialized) {
+			fetchingServers = true;
+			try {
+				servers = await getServers();
+			} catch (e) {
+				await emit('error', e);
+			}
+			fetchingServers = false;
+		}
+	};
+
+	const handleServerCreate = async () => {
+		if (selected === null) {
+			return;
+		}
+
+		try {
+			await updateServers();
+			selected = get(selectedCommit);
+		} catch (e) {
+			await emit('error', e);
+		}
+	};
+
+	const handleSyncClient = async (entry: Nullable<ArtifactEntry>, server: GameServerResult) => {
```

Comment:
i think this `handleSyncClient` may be unnecessary. it's used in handleAutoLaunch below, but it's not entirely clear to me why that autolaunch can't be moved into the modal.